packages/web-components/src/counter-badge/counter-badge.template.ts

```diff
@@ -0,0 +1,16 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import { badgeTemplate } from '../badge/badge.template';
```

Comment:
```suggestion
import { badgeTemplate } from '../badge/badge.template.js';
```
?

Reply:
I'm finding that VSCode doesn't consistently add these - I think it could be something with nested tsconfig settings. Not intentional 👍